tgui/packages/tgui/interfaces/BotClean.js

```diff
@@ -0,0 +1,97 @@
+import { Fragment } from 'inferno';
+import { useBackend } from '../backend';
+import { Button, LabeledList, NoticeBox, Section, Box } from '../components';
+import { Window } from '../layouts';
+
+export const BotClean = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    locked,
+    noaccess,
+    maintpanel,
+    on,
+    autopatrol,
+    canhack,
+    emagged,
+    remote_disabled,
+    painame,
+
+    cleanblood,
+  } = data;
+  return (
+    <Window>
+      <Window.Content scrollable>
+        <NoticeBox>
+          Swipe an ID card to {locked ? 'unlock' : 'lock'} this interface.
+        </NoticeBox>
+        <Section title="General Settings">
+          <LabeledList>
+            <LabeledList.Item label="Status">
+              <Button
+                icon={on ? 'power-off' : 'times'}
+                content={on ? 'On' : 'Off'}
+                selected={on}
+                disabled={noaccess}
+                onClick={() => act('power')} />
+            </LabeledList.Item>
+            <LabeledList.Item label="Patrol">
+              <Button.Checkbox
+                fluid
+                checked={autopatrol}
+                content="Auto Patrol"
+                disabled={noaccess}
+                onClick={() => act('autopatrol')} />
+            </LabeledList.Item>
+            {!!maintpanel && (
+              <LabeledList.Item label="Maintenance Panel">
+                <Box color="bad">
+                  Panel Open!
+                </Box>
+              </LabeledList.Item>
+            )}
+            <LabeledList.Item label="Safety System">
+              <Box color={emagged ? 'bad' : 'good'}>
+                {emagged ? "DISABLED!" : "Enabled"}
+              </Box>
+            </LabeledList.Item>
+            {!!canhack && (
+              <LabeledList.Item label="Hacking">
+                <Button
+                  icon="terminal"
+                  content={emagged ? "UnHack" : "Hack"}
```

Comment:
```suggestion
                  content={emagged ? "Unhack" : "Hack"}
```
Maybe even `Restore` or `Revert`, something about "Unhack" just sounds off, and the spelling of UnHack is definitely off.